DataFormats/Math/test/ExtVec_t.cpp

```diff
@@ -1,25 +1,21 @@
-#include "FWCore/Utilities/interface/GCC11Compatibility.h"
-#if GCC_PREREQUISITE(4,8,0)
-
 #include "DataFormats/Math/interface/ExtVec.h"
 
 #include<cmath>
 #include<vector>
 
 #include<iostream>
 
+#ifdef __AVX__
+#define CMS_USE_AVX
+#endif /* __AVX__ */
 
-// #include <x86intrin.h>
+// clang complains that it is not "standard"
+#define constexpr
```

Comment:
btw the current combination of clang 3.8, gcc 6.3 and std=c++1z is completely bogus:
does not link.
one needs either clang4.0, and std=c++14 or move to gcc7...